test/snapshots/markdownlint-test-scenarios.mjs.md

```diff
@@ -16178,6 +16178,87 @@ Generated by [AVA](https://avajs.dev).
       `,
     }
 
+## heading_increment-alt_title_level3_ko.md
```

Comment:
As with your other PR, please revert all snapshot changes other than the one for the scenarios tests.